tests/auth_tests/test_auth_backends.py

```diff
@@ -612,13 +616,41 @@ class ImportedBackendTests(TestCase):
     """
 
     backend = 'auth_tests.backend_alias.ImportedModelBackend'
+    other_backend = 'auth_tests.backend_alias.OtherImportedModelBackend'
+    username = 'username'
+    password = 'password'
 
     @override_settings(AUTHENTICATION_BACKENDS=[backend])
     def test_backend_path(self):
-        username = 'username'
-        password = 'password'
-        User.objects.create_user(username, 'email', password)
-        self.assertTrue(self.client.login(username=username, password=password))
+        User.objects.create_user(self.username, 'email', self.password)
+        self.assertTrue(self.client.login(username=self.username, password=self.password))
+
+        self._expect_backend_in_session(self.backend)
+
+    @override_settings(AUTHENTICATION_BACKENDS=[backend])
+    def test_backend_path_login_without_authenticate_single_backend(self):
+        user = User.objects.create_user(self.username, 'email', self.password)
+
```

Comment:
You can skip the blank lines in these tests as three lines long isn't too difficult too read with spaces.